chartpress-gitlab.yaml

```diff
@@ -0,0 +1,25 @@
+# this file should be used as and when necessary to build the renku gitlab helm
+# chart. As renku moves to use of the standard gitlab cloud native chart, this
+# will become redundnat but for now it is still in use.
+#
+# To use this, you will need to remove the chartpress.yaml for building renku and
```

Comment:
this seems sub-optimal - can we contribute a `--file` option to the upstream?

Reply:
Sounds v doable - prob just adding an arg with a default and prob a test.

I can look into it.

Reply:
I made https://github.com/jupyterhub/chartpress/issues/170